sheet_to_triples/field.py

```diff
@@ -40,7 +40,7 @@
 
 def _must(value, exception_type=ValueError):
     """Raise exception_type if value is falsey."""
-    if not value:
+    if not value or (isinstance(value, str) and not value.strip()):
```

Comment:
Must shouldn't really futz with string things, callers should do things to the value before if they need.